change/@fluentui-react-spinner-a46a30d1-435e-4315-9799-2cdaa4a8579f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32989&commitId=2a31b28af940142ffaf0d46166c3347903a53ebb&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|14| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|2| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2232| Changed|
|Drawer.overlay drawer full.chromium.png|3285| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>